Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -474,7 +487,8 @@ def revoke_user_session_command(client: MsGraphClient, args: dict):
 
 def main():
     params: dict = demisto.params()
-    url = params.get('host', '').rstrip('/') + '/v1.0/'
+    azure_cloud = get_azure_cloud(params, 'MicrosoftGraphUser')
+    url = urljoin(azure_cloud.endpoints.microsoft_graph_resource_id.rstrip("/"), '/v1.0/')
```

Comment:
```suggestion
    url = urljoin(azure_cloud.endpoints.microsoft_graph_resource_id, '/v1.0/')
```
maybe add constant: API_VERSION: str = 'v1.0'
like in MicrosoftGraphDeviceManagement.py